java/code/src/com/redhat/rhn/manager/channel/CreateChannelCommand.java

```diff
@@ -44,9 +44,9 @@
     public static final int CHANNEL_NAME_MAX_LENGTH = 256;
     public static final int CHANNEL_LABEL_MIN_LENGTH = 6;
 
-    protected static final String CHANNEL_NAME_REGEX =
+    public static final String CHANNEL_NAME_REGEX =
```

Comment:
What about to remove the `CHANNEL_` prefix and keep only the variable name like `LABEL_REGEX` and `NAME_REGEX`? I see it is used as well for the project, so maybe we should keep it more abstract. I would also move the two variables to the `ValidationUtils.java` class instead, as they belong more to it and they can be reused from any other `Create<Entity>Command.java` class as well maybe. What do you think?

Reply:
I am not just it makes sense to generalize that, they are still Channel_Name_Regex and not general labels. We need to check them just because the project label will be part of the cloned channels

Reply:
I see. I think we have the same restrictions for every `label` we save in the database, but maybe it's not, and btw it is not the goal of this PR, so it is fine to keep them a *channel-specific* regex. :+1: